CIP-1853/CIP-1853.md

```diff
@@ -0,0 +1,58 @@
+---
+CIP: 1853
+Title: HD (Hierarchy for Deterministic) Stake Pool Cold Keys for Cardano
+Authors: Rafael Korbas <rafael.korbas@vacuumlabs.com>
+Comments-URI: https://forum.cardano.org/t/stake-pool-cold-keys-hd-derivation/43360
+Status: Draft
+Type: Standards
+Created: 2020-12-14
+License: CC-BY-4.0
+---
+
+## Abstract
+
+[CIP-1852](https://github.com/cardano-foundation/CIPs/blob/master/CIP-1852/CIP-1852.md) establishes how Shelley-era hierarchical deterministic (HD) wallets should derive their keys. This document is a follow-up of this CIP specifying how stake pool cold keys should be derived.
+
+## Motivation
+
+(Hierarchical) deterministic derivation of stake pool cold keys enables their restorability from a seed and most importantly, their management on hardware wallet devices. This in turn mitigates man-in-the middle attacks to which pool operators would otherwise be vulnerable if they managed their stake pool cold keys on a device not specifically hardened against alteration of the data to be signed/serialized without operator's explicit consent.
+
+## Specification
+
+Using `1853'` as the purpose field, we define the following derivation path structure for stake pool cold keys.
+
+```
+m / purpose' / coin_type' / usecase / cold_key_index'
+```
+
+Example: `m / 1853' / 1815' / 0 / 0'`
+
+Here the `usecase` is currently fixed to `0`.
```

Comment:
I agree in the future some use-cases may want to be soft-derived.
I'm not sure if there is any reason to have this specific use case be soft-derived instead of hard-derived though. given all child keys are hard-derived.